mbuild/compound.py

```diff
@@ -1009,6 +1010,65 @@ def generate_bonds(self, name_a, name_b, dmin, dmax):
                     self.add_bond((p1, p2))
                     added_bonds.append(bond_tuple)
 
+    @experimental_feature()
+    def freud_generate_bonds(
+        self,
+        name_a,
+        name_b,
+        dmin,
+        dmax,
+        exclude_ii=True,
+    ):
+        """Add Bonds between all pairs of types a/b within [dmin, dmax].
+
+        Parameters
+        ----------
+        name_a : str
+            The name of one of the Particles to be in each bond
+        name_b : str
+            The name of the other Particle to be in each bond
+        dmin : float
+            The minimum distance between Particles for considering a bond
```

Comment:
do you think we should mention the unit here to (add `in nm` at the end of the sentence)?

Reply:
Good catch, i will add that in now